### PR TITLE
Add disaggregated analysis table to Model Overview

### PR DESCRIPTION
**Pull request by romanlutz**
## Description

The title says it all 🙂 Other than the disaggregated analysis I'm also adding textured color for NaN values and changing the line wrapping code slightly. We now wrap on the last whitespace in a line (or at 40 characters), and all the labels go into a grid (as an extension to the table).

See the video below for a quick demo:

https://user-images.githubusercontent.com/10245648/163058848-1e8c5472-6653-4ad7-846b-2320d72f4a7e.mp4

## Areas changed

npm packages changed:

- [ ] responsibleai/causality
- [x] responsibleai/core-ui
- [ ] responsibleai/counterfactuals
- [ ] responsibleai/dataset-explorer
- [ ] responsibleai/fairness
- [ ] responsibleai/interpret
- [x] responsibleai/localization
- [ ] responsibleai/mlchartlib
- [x] responsibleai/model-assessment

Python packages changed:

- [ ] erroranalysis
- [ ] raiutils
- [ ] raiwidgets
- [ ] rai_core_flask
- [ ] responsibleai

## Tests

Tests will be added before feature flag is removed, in a different PR.

- [ ] No new tests required.
- [ ] New tests for the added feature are part of this PR.
- [ ] I validated the changes manually.

## Screenshots (if appropriate):

see video above for reference

## Documentation:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by tongyu-microsoft — APPROVED** [review]
UI styling question on feature selection dropdown from the demo video: do we want to add scroll bar for dropdown list, as the feature list might be really long? Should we limit Height of the dropdown list? Similar like how we currently do for counterfactual datapoint selection dropdown (attached screenshots below):
image

image

**Comment by minthigpen** [comment]
> @romanlutz UI styling question on feature selection dropdown from the demo video: do we want to add scroll bar for dropdown list, as the feature list might be really long? Should we limit Height of the dropdown list? Similar like how we currently do for counterfactual datapoint selection dropdown (attached screenshots below): image
> 
> image

Yes we want to do a scroll bar and limit the height of the dropdown list. If we can make the drop down experience consistent across the RAI dashboard that would be the best.

**Comment by romanlutz** [comment]
Added this using `FabricStyles` which has the entire configuration:
image

I've applied it to both dropdown (now combo boxes), but the metric selection doesn't have enough elements to require the scrollbar. 🙂

File: libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/StatsTableUtils.ts [hunk 1]
```diff
@@ -65,9 +65,24 @@ export function generateCohortsStatsTable(
             y: cohortIndex
           });
         } else {
-          // not a numeric value (NaN), so just put null and use color gray
+          // not a numeric value (NaN), so just put null and use textured color
           items.push({
-            color: "#808080",
+            color: {
```
Comment (imatiach-msft) [inline libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/StatsTableUtils.ts:70]: oh cool!!!

**Review by imatiach-msft — APPROVED** [review]

File: libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverview.tsx [hunk 2]
```diff
@@ -120,42 +125,47 @@ export class ModelOverview extends React.Component<
...
+      >
         <Text variant="medium" style={{ maxWidth: descriptionMaxWidth }}>
           {localization.Interpret.ModelPerformance.helperText}
         </Text>
         {!this.props.showNewModelOverviewExperience && <OverallMetricChart />}
         {this.props.showNewModelOverviewExperience && (
           <>
             <Stack horizontal tokens={{ childrenGap: "20px" }}>
-              <Dropdown
+              <ComboBox
```
Comment (imatiach-msft) [inline libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverview.tsx:139]: What happens when you choose a ton of features, say 100 features - is the first column still readable in that case?  Does the UI break in any way?

Comment (imatiach-msft) [reply libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverview.tsx:139]: Or do we impose some limit on the number of features the user can choose in the dropdown?

Comment (romanlutz) [reply libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverview.tsx:139]: This is the metrics dropdown. You can choose as many as you like, but there are only like 7 for binary classification that we currently support. Perhaps more in the future.

For the feature dropdown you can only choose two, at which point the other options get disabled. You can see the video for an example (see PR description).

File: libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/StatsTableUtils.ts (hunk 1 above)
Comment (romanlutz) [reply libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/StatsTableUtils.ts:70]: 🙂

**Pull request merged**